wishlist.graphql

```diff
@@ -123,22 +123,14 @@ type ProductVariantConnection @shareable {
 
 type Query {
 	"""
-	Retrieves all wishlists.
+	Retrieve user with wishlists.
 	"""
-	wishlists(
-		"""
-		Describes that the `first` N wishlists should be retrieved.
-		"""
-		first: Int,
-		"""
-		Describes how many wishlists should be skipped at the beginning.
+	user(
```

Comment:
and why is there a user query?
you should have a resolver, but not the query

Reply:
So, what do we do with this PR then?